internal/build_message.ts

```diff
@@ -112,3 +111,9 @@ export function buildMessage(
   messages.push(...(stringDiff ? [diffMessages.join("")] : diffMessages), "");
   return messages;
 }
+
+/** Used internally for testing. */
+export const _internals = {
```

Comment:
Let's export these functions instead of putting them in these `_internals` objects. It'll be much cleaner. Ditto for the other files in this package. This is fine to do here as this is purely an internal package. You'll have to add `@example` tags to the newly exported symbols to make the doc checker (`deno task lint:docs`) too.

Reply:
gotcha!
for some reason i was unsure if we would ever want to export the other functions in this package.
implemented as suggested.
love the way how the doc checker works bth!

Reply:
Are you able to fix the functions in this file up too?

Reply:
oh... i'm so sorry.
cleanup for `build_message` is done as well.